marketplace/google-analytics/src/app.js

```diff
@@ -0,0 +1,135 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import {Timeline} from './timeline.js';
+import {formatDate} from './utils.js';
+
+const DAY_IN_MS = 1000 * 60 * 60 * 24;
+const TODAY = new Date();
+const TIMELINE_DIMENSIONS = [
+  {label: 'Day', value: 'date'},
+  {label: 'Week', value: 'week'},
+  {label: 'Month', value: 'month'}
+];
+const CLIENT_ID = '318721834234-s3td95ohvub1bkksn3aicimnltvmtts8.apps.googleusercontent.com';
```

Comment:
Shouldn't it be a parameter?

Reply:
Why? That would require the user to create their own oAuth app. This way it's already done which saves _a lot_ of hassle setting up the app.